### PR TITLE
[UPD]composer.json: composer create-project installing older version

### PR DESCRIPTION
**Pull request by christer77**
[related issue](https://github.com/cypht-org/cypht/issues/940)

### CODE REVIEW

**Review by Shadow243 — CHANGES_REQUESTED** [review]
I actually see version 4.17 with bug fixes and some good reactions. Seems like they're making steady improvements.

[https://github.com/ezyang/htmlpurifier/releases](https://github.com/ezyang/htmlpurifier/releases)

**Comment by marclaporte** [comment]
@Shadow243 Since composer.json has "ezyang/htmlpurifier": "^4.16", it should get 4.17, 4.18, etc. right?

**Comment by marclaporte** [comment]
Yes, we are already getting: https://github.com/cypht-org/cypht/blob/master/composer.lock#L249

@christer77 I don''t understand how this MR fixes the issue

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "ext-mbstring": "*",
         "ext-openssl": "*",
         "ext-session": "*",
-        "ezyang/htmlpurifier": "^4.13",
+        "ezyang/htmlpurifier": "^4.16",
```
Comment (Shadow243) [inline composer.json:35]: @christer77  Can you add composer.lock to this PR please!

File: composer.lock
Comment (marclaporte) [inline composer.lock:1]: ezyang/htmlpurifier doesn't change in composer.lock

**Pull request merged**